pkg/apis/configs/v1alpha1/configmappropagation_lifecycle.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"knative.dev/pkg/apis"
+)
+
+var configMapPropagationCondSet = apis.NewLivingConditionSet(ConfigMapPropagationConditionReady, ConfigMapPropagationConditionPropagated)
+
+const (
+	// ConfigMapPropagationConditionReady has status True when all subconditions below have been set to True.
+	ConfigMapPropagationConditionReady = apis.ConditionReady
+	// ConfigMapPropagationConditionPropagated has status True when the ConfigMaps in original namespace are all propagated to target namespace.
+	ConfigMapPropagationConditionPropagated apis.ConditionType = "ConfigMapPropagated"
```

Comment:
Does the condition need to include "ConfigMap", since it applies to a "ConfigMapPropagation"?

Reply:
Could you explain more about this? (what kind of "ConfigMap" condition do you think that can be included)

Reply:
I'm just talking about the string, I was just suggesting that "Propagated" would be less redundant "ConfigMapPropagated".